app/model/Skautis/WebserviceFactory.php

```diff
@@ -4,33 +4,36 @@
 
 namespace Model\Skautis;
 
+use Model\Infrastructure\Skautis\LazyWebService;
 use Nette\Caching\IStorage;
 use Nette\StaticClass;
 use Skautis\Nette\CacheAdapter;
 use Skautis\Skautis;
 use Skautis\Wsdl\Decorator\Cache\ArrayCache;
 use Skautis\Wsdl\Decorator\Cache\CacheDecorator;
-use Skautis\Wsdl\WebService;
 
+/**
+ * @see LazyWebService
+ */
 final class WebserviceFactory
 {
     use StaticClass;
 
     public static function createCached(string $name, Skautis $skautis): CacheDecorator
     {
-        return new CacheDecorator($skautis->getWebService($name), new ArrayCache());
+        return new CacheDecorator(self::create($name, $skautis), new ArrayCache());
     }
 
     public static function createOrg(Skautis $skautis, IStorage $storage): CacheDecorator
```

Comment:
Tohle se podle mě nikde neupožívá, tak bych to rovnou odebrat taky

Reply:
Odebral jsem